[docs] Update usage permissions (#380)
seqeralabs/tower-cli: USAGE.md

## Pipelines

> A Pipeline is composed of a workflow repository, launch parameters, and a Compute Environment. Pipelines are used to define pre-configured workflows in a Workspace.

Author: ```suggestion
A pipeline consists of a workflow repository, launch parameters, and a Compute Environment. Pipelines are used to define pre-configured workflows in a Workspace.
```

> **NOTE**: The `params-file` option should be a YAML or JSON file.

### Importing/exporting a pipeline

You can export the configuration details of a pipeline in JSON format for scripting and reproducibility purposes.

```console
$ tw pipelines export --name=my_rnaseq_nf_pipeline my_rnaseq_nf_pipeline_v1.json

  Pipeline exported into 'my_rnaseq_nf_pipeline_v1.json' 
```

Similarly, a pipeline can easily be imported into a Workspace from a previously exported JSON file.

```console
$ tw pipelines import --name=my_rnaseq_nf_pipeline_v1 ./my_rnaseq_nf_pipeline_v1.json

  New pipeline 'my_rnaseq_nf_pipeline_v1' added at user workspace
```

### Updating a pipeline

The default launch parameters can be changed using the `update` command:

```console
tw pipelines update --name=my_rnaseq_nf_pipeline --params-file=my_rnaseq_nf_pipeline_params_2.yaml
```

## Launching pipelines

### Launching a preconfigured pipeline

When launching a pipeline from the Launchpad, if no custom parameters are passed via the CLI then the defaults set for the pipeline in the Launchpad will be used.

> > **NOTE**: Platform CLI users are bound to the same user permissions that apply in the platform UI. Launchpad users can launch pre-configured pipelines in the workspaces they have access to, but they cannot add or run new pipelines. 

Author: ```suggestion
> **NOTE**: Platform CLI users are bound to the same user permissions that apply in the platform UI. Launch users can launch pre-configured pipelines in the workspaces they have access to, but they cannot add or run new pipelines. 
```

    https://tower.nf/user/abhinav/watch/2XDXxX0vCX8xhx
```

> > **NOTE**: Platform CLI users are bound to the same user permissions that apply in the platform UI. Launchpad users can launch pre-configured pipelines in the workspaces they have access to, but they cannot add or run new pipelines. 

Author: ```suggestion
> **NOTE**: Platform CLI users are bound to the same user permissions that apply in the platform UI. Launch users can launch pre-configured pipelines in the workspaces they have access to, but they cannot add or run new pipelines. 
```
